packages/core/src/Site/index.js

```diff
@@ -437,13 +437,13 @@ class Site {
       .forEach((page) => {
         const addressablePagePath = path.join(this.rootPath, page.src);
         const relativePagePathWithoutExt = fsUtil.removeExtension(
-          path.relative(this.rootPath, addressablePagePath));
+          path.relative(this.rootPath, addressablePagePath)).replace(/\\/g, '/');
```

Comment:
we have a `fsUtil.ensurePosix` wrapper call for this

Since this change is also distinct from the other issue, let's organise this PR into 2 commits (or preferably, another PR) as well, so its clear these 2 problems aren't related:
- fix issue in this line
- fix issue in 446

I'll be doing a rebase (without squash) of this PR onto the main branch if you opt for the former!

Reply:
I choose to keep these 2 in the same PR since if I split the PR into 2 different commits the tests will fail, so perhaps I should try organizing this into 2 commits? Do you have any recommendations on how to achieve this - is it  done by using rebase (but wont it still show up here)?

Reply:
Ok I think we can ditch this PR, seems like more work to try and fix all the issues with it (like all the extra unneeded changes). I will first make a PR for the backslash error, then make another PR for the newline fix with all the test cases updated. Does that sound good?

Reply:
That works too! Especially since the changes here are small.

More generally however if replicating the changes becomes infeasible, you could rebase the branch on top of the main branch, then perform a soft reset. (then recommit just the changes for line 446 in this pr)